python/oneseismic/login/__main__.py

```diff
@@ -73,7 +73,7 @@ def main(argv):
     parser.add_argument('url',
             type  = str,
             nargs = '?',
-            help  = 'Base URL to get oneseismic config from. '
+            help  = 'URL to get oneseismic config from. '
```

Comment:
This argument does in fact expect a base url. The /config suffix is added in the assumehttps function.